templates/etc/squid/squid.conf.j2

```diff
@@ -31,7 +39,11 @@ acl Safe_ports port {{ item['port'] }}{% if item['comment'] is defined %} #{{ it
 
 {% if squid_http_access is defined and squid_http_access != [] %}
 {%   for item in squid_http_access %}
+{% if 'domain_blacklist' in item['acl'] and (squid_domain_blacklist_file is not defined or squid_domain_blacklist == []) %}
+{% elif 'domain_whitelist' in item['acl'] and (squid_domain_whitelist_file is not defined or squid_domain_whitelist == []) %}
+{% else %}
 http_access {{ item['action'] }} {{ item['acl']|join(' ') }}
```

Comment:
I don't quite understand the interaction between acl and http_access

Reply:
Here's the syntax:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/configuring-a-domain-blacklist-in-squid